Licel/TCP_util.py

```diff
@@ -90,4 +90,4 @@ def _writeReadAndVerify(self, command, verifyString):
         resp= self.readResponse()
         if resp.find(verifyString) == -1 :
             raise RuntimeError(resp)
-        return resp
\ No newline at end of file
+        return resp
```

Comment:
This change seems to be unrelated to the addition of the project file. Can we move it to a new branch? I wouldn't be suprised if there are even more files that suffer from this missing newline.

Reply:
My text editor automatically adds new line at the end of the file if it's missing, reverted that change.